### ISSUE TITLE
ClickHouse error: `Cannot parse IPv6 ?????`

### ISSUE THREAD

**Issue by vincentbernat** [akvorado/akvorado#717]
### Discussed in https://github.com/akvorado/akvorado/discussions/698

<div type='discussions-op-text'>

<sup>Originally posted by **MattKobayashi** May 15, 2023</sup>
Hi all,

I'm having trouble with my Akvorado instance not displaying received/processed flows. I've followed the troubleshooting processes in the documentation, and determined that:

- Akvorado is receiving the flows from the exporter (MikroTik CCR2004 running RouterOS v7.9) and sending them to Kafka
- Kafka is sending the flows through to ClickHouse
- ClickHouse is receiving the flows from Kafka

However, ClickHouse is displaying an odd error:

```
Row 2:
──────
last_error_time:    2023-05-15 04:51:05
last_error_message: Cannot parse IPv6 ?????
```

I suspect that the MikroTik is exporting something weird for IPv6 flows that ClickHouse isn't recognising. Is anyone able to assist with troubleshooting this further?</div>

**Comment by vincentbernat** [comment]
This does not work with ClickHouse 23.3. Using just `IPv6` instead of `LowCardinality(IPv6)` for `ExporterAddress` does not help.

**Comment by vincentbernat** [comment]
See https://github.com/ClickHouse/ClickHouse/issues/49924

**Comment by vincentbernat** [comment]
This is fixed. Waiting for the next LTS to be released.

**Comment by vincentbernat** [comment]
New LTS is released and there is now a version check in 87be0ed37486.
